src/main/scala/com/gu/newsletterlistcleanse/GetCutOffDatesLambda.scala

```diff
@@ -21,7 +21,7 @@ object GetCutOffDatesLambda {
   /*
    * This is your lambda entry point
    */
-  def handler(lambdaInput: GetCleanseListLambdaInput, context: Context): Unit = {
+  def handler(lambdaInput: GetCutOffDatesLambdaInput, context: Context): Unit = {
```

Comment:
Is this lambda redundant now? GetCleanseList seems to calculate the cutoff dates

Reply:
It looks like the incorrect Lambdas have been used here actually. The current logic should all have gone in the `GetCutOffDates` lambda